web/views.py

```
@@ -6341,3 +6408,492 @@ def delete_post(request, post_id):
     if request.method == "POST":
         post.delete()
     return redirect("social_media_dashboard")
+
+
+@login_required
+def pre_session_checkin(request, session_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `pre_session_checkin`

(ANN201)